external-crates/move/crates/move-vm-runtime/src/execution/values/values_impl.rs

```diff
@@ -2673,7 +2673,6 @@ pub mod debug {
         Ok(())
     }
 
-    // TODO: This function was used in an old implementation of std::debug::print, and can probably be removed.
     pub fn print_reference<B: Write>(buf: &mut B, r: &Reference) -> PartialVMResult<()> {
```

Comment:
Can we remove this function how?

Reply:
I had a look, and we can't -- this is used in `print_value` which is used in the test framework + stack trace printing. So I don't think we can unless we want to remove those callsites which seems like something we probably don't want to do.